src/python/bot/tasks/update_task.py

```diff
@@ -171,7 +171,8 @@ def run_platform_init_scripts():
 
   plt = environment.platform()
   if environment.is_android():
-    android_init.run()
+    if not environment.is_android_emulator():
```

Comment:
Forgot to ask, why are we doing this, this bailout needs to be added in the specific init function call, like what breaks with emulator ? Is it this ? https://github.com/google/clusterfuzz/blob/4290129370d0d3c6198ae752e4e4eff2b59621a6/src/python/bot/init_scripts/android.py#L25

maybe bailout needs to be added at start of flash_to_latest_build_if_needed ?
basically some functions in that android_init might be useful, like restarting device? so we shouldn't just skip it all.

Reply:
Ok!